controlpanel/settings/common.py

```diff
@@ -252,9 +252,9 @@
 DATABASES = {
     "default": {
         "ENGINE": "django.db.backends.postgresql",
-        "NAME": os.environ.get("DB_NAME", PROJECT_NAME),
```

Comment:
This one has a default

Reply:
I know but we pass them anyway and I think we should try and be explicit as we have to pass the rest anyway.